maestrowf/datastructures/schemas.json

```diff
@@ -0,0 +1,118 @@
+{
+  "DESCRIPTION": {
+    "type": "object",
+    "properties": {
+      "name": {"type": "string", "minLength": 1},
+      "description": {"type": "string", "minLength": 1}
+    },
+    "additionalProperties": false,
```

Comment:
Just a quick comment -- I was testing this and found that this should probably be set to `true`. The `description` block is meant to have other types of information that isn't necessarily `name` and `description`. Just a comment, I didn't think this was worth un-approving the PR.